CI/hf_mirror/download_hf_models.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+df -h  # debug purpose to make sure efs is attached correclty
```

Comment:
[minor] We can fail the script if the volume is not attached correctly with some error message which can be more indicative for users

Reply:
good point